### DISCUSSION TITLE
How-to pull/sync existing vault on the new device?

### DISCUSSION THREAD

**Discussion by craziedde** [remotely-save/remotely-save#880] [Q&A]
Good afternoon,

How-to pull/sync existing vault on the new device?
I seem to missing this information

@craziedde

**Comment by fyears** [comment]
create an empty vault in obsidian with the same vault name, then install remotely save and configure it, then sync

**Reply by craziedde** [reply]
> create an empty vault in obsidian with the same vault name, then install remotely save and configure it, then sync

yes, it worked files synced.
no configurations & no plugins synced?

@craziedde
